common/lib/xmodule/xmodule/modulestore/split_mongo/split_mongo_kvs.py

```diff
@@ -148,7 +148,10 @@ def has(self, key):
             if key.scope == Scope.children:
                 return False
 
-            b_type = key.block_scope_id.block_type
+            try:
+                b_type = key.block_scope_id.block_type
```

Comment:
@cpennington generally (for example for [StructuredTagsAside](https://github.com/edx/edx-platform/blob/master/cms/lib/xblock/tagging.py#L68)) `key.block_scope_id` is an object of `AsideUsageKeyV1` type. But there is one [dependency](https://github.com/edx/edx-platform/blob/master/requirements/edx/github.txt#L77) - it named `AcidAside`. I don't know why but in case of `AcidAside` object `block_scope_id` is always string:
![pycharm](https://cloud.githubusercontent.com/assets/1876893/13797247/282c629a-eb1d-11e5-950f-afed18a2e71f.jpeg)